book/website/communication/citable/citable-cff.md

```diff
@@ -0,0 +1,153 @@
+
+#### Step 3. Add Your `CITATION.cff` to a Public Code Repository
+
+After creating a `CITATION.cff` file, you will need to add it to your GitHub repository so that GitHub can recognize it. GitHub already has instructions to show how to do this.For instructions please see [GitHub's guide on software citation.](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files)
+
+(cm-citable-cff-how-to-cite)=
+## How to Cite Using `CITATION.cff` (Abel)
+
+If you have found a software or dataset that contains a `CITATION.cff`, there are a few ways to obtain the reference information to cite it in your publication.
+
+- You can use one of the tools, such as the `cffconvert` command line program, to convert your `CITATION.cff` file to one of the [supported formats](https://github.com/citation-file-format/cff-converter-python#supported-output-formats), such as APA, BibTeX or EndNote.
+
+- Alternatively, if the software or dataset you want to cite is available on GitHub, you can use GitHub's interface to copy the reference in either APA or BibTeX formats by clicking the "Cite this repository" button (see the green area in the image below).
+
+  ```{note}
+  "Cite this repository" button only appears when there is a `CITATION.cff` file on the repository.
+  ```
+
+  ```{figure} ../../figures/github-cff-integration.jpg
+  ---
+  name: github-cff-integration
+  alt: Button on GitHub that provides automatically conversion of the `CITATION.cff` file to APA's and BibTex's format.
+  ---
+  "Cite this repository" automatically converts the `CITATION.cff` file to APA's and BibTex's format.
+  ```
+
+(cm-citable-cff-available-tools)=
+## Available Tools (Abel and Faruk)
+
+Several tools exist to facilitate the creation and validation of `CITATION.cff` files, as well as the conversion to and from other formats.
+See the [table at the repository's specification](https://github.com/citation-file-format/citation-file-format#tools-to-work-with-citationcff-files-wrench) for a list of all known tools.
+
+<!--
+Below we list a few of the main ones.
+
+### cffinit
+
+The web application [cffinit](https://citation-file-format.github.io/cff-initializer-javascript/) provides an interactive way of creating a `CITATION.cff` file online.
+The app provides a simple interface to create your first `CITATION.cff` file.
+You can use it to add the bare minimum of information or you can add optional, but very important, fields, like the DOI, code repository, release date, etc.
+This approach is recommended due to having validation built-in.
+
+### cffconvert
+
+[cffconvert](https://github.com/citation-file-format/cff-converter-python) is a command line program to validate and convert `CITATION.cff` files.
+It can be used to convert to other formats, like BibTex, EndNote, and Zenodo, as well as validate `CITATION.cff` files locally or in GitHub repositories.
+-->
```

Comment:
I'm guessing you'll leave this out if you include the separate section on `cffinit`, right?

Reply:
this was already removed.